### ISSUE TITLE
[website] GitHub button is not working

### ISSUE THREAD

**Issue by goenning** [ardeora/solid-toast#5]
Hey, I couldn't find the source code of the https://www.solid-toast.com/ otherwise I'd have sent a PR.

But just letting you know that the GitHub button doesn't work.

Great work on solid-toast! 🎉

**Comment by ardeora** [comment]
Hey there! Thanks for the kind words :D The website hasn't been completed yet. I should have put a banner explicitly saying that. So I apologize for the confusion there. I was planning on making an Astro site that shows different toast examples and provides full documentation. I'll create a skeleton for it this weekend and add it to this repo. I would love to have more contributors to improve `solid-toast` and the website. I'll keep you posted on this issue.

**Comment by goenning** [comment]
@ardeora by the way, where does the `animate-enter` and `animate-leave` come from on your Tailwind CSS example? Doesn't seem to be from tailwindcss and I can't find the source code either :)

**Comment by ardeora** [comment]
Yeah apologies for that. I'm planning on making the website public as well but it the code for it is a little dirty. I'll try to clean it up this week but here is my tailwind config that set us up the enter and exit animations if you're interested in it.


[Tailwind Config For Solid Toast](https://gist.github.com/ardeora/0ce988b30b750eb32ec48a5a6431e768)

**Comment by goenning** [comment]
Awesome! Thank you! 

The website is really good! Great work! 🎉
